views/error.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Chitter: Oops!</title>
+  </head>
+    <center>
+  <header>
+    <%= erb(:title) %>
```

Comment:
This is a very clean way to have an header in each page you want it. It respect the DRY concept.